app/ide-desktop/lib/dashboard/src/authentication/src/authentication/service.tsx

```diff
@@ -190,3 +206,46 @@ const setDeepLinkHandler = (
  * email, then the URL will be for the confirmation page path. */
 const isConfirmRegistrationRedirect = (url: URL) =>
   url.pathname === CONFIRM_REGISTRATION_PATHNAME;
+
+/** If the user is being redirected after a sign-out, then query args will be present. */
+const isSignInRedirect = (url: URL) =>
+  url.pathname === SIGN_IN_PATHNAME && url.search !== "";
```

Comment:
again we have semicolons in the code. I was writing about it in previous PR - did something change in our prettier config? We should not have semicolons.

Reply:
prettier config still doesn't include .jsx/.tsx files - i think it'll be easier to just merge up to 7/7 at least, and then format everything afterwards

Reply:
Agreed w/ @somebody1234. Applying the prettier config now creates massive merge conflicts.

Reply:
I forgot about this plan! Ok :)